report/src/presence_technologies/bluetooth.tex

```diff
@@ -0,0 +1,44 @@
+\section{Bluetooth} \label{sec:bluetooth}
+
+Another possible technology for presence detection is Bluetooth.
+Bluetooth is a common technology used in many devices such as smartphones, tablets, and laptops.
+Because it is a standardized technology, all devices that support Bluetooth can communicate with each other.
+
+There are two different types of radio signals used in Bluetooth: Classic and Low Energy (LE).
+They differ in terms of power consumption, range, and data transfer rate.\cite{BluetoothSIG}
+
+\begin{figure}[h!]
+  \centering
+  \includegraphics[width=1\textwidth]{Bluetooth_Technology_Overview_Graphic.png}
+  \caption{Overview of the Bluetooth Classic and Low Energy technologies\cite{BluetoothOverview}.}
```

Comment:
They might have a download button, but does that mean that we can use the picture? Ask Aebeloe.